test/functional/test-fixed-layer.js

```diff
@@ -1339,83 +1339,83 @@ describes.realWin('FixedLayer: shadow transfer', {}, env => {
   let shadowRoot;
   let container;
 
-  beforeEach(function() {
-    if (!env.win.Element.prototype.attachShadow) {
-      // Can only test when SD is supported.
-      this.skip();
-    }
-    win = env.win;
-    doc = win.document;
-    vsyncTasks = [];
-    vsyncApi = {
-      runPromise: task => {
-        vsyncTasks.push(task);
-        return Promise.resolve();
-      },
-      mutate: mutator => {
-        vsyncTasks.push({mutate: mutator});
-      },
-    };
-    installPlatformService(win);
-    ampdoc = new AmpDocSingle(win);
-    shadowRoot = win.document.body.attachShadow({mode: 'open'});
-    fixedLayer = new FixedLayer(ampdoc, vsyncApi,
-        /* borderTop */ 0, /* paddingTop */ 11, /* transfer */ true);
-    fixedLayer.setup();
-    transferLayer = fixedLayer.getTransferLayer_();
-    root = transferLayer.getRoot();
-    container = doc.createElement('div');
-    doc.body.appendChild(container);
-  });
+  // Can only test when SD is supported.
+  describe.configure().if(() => Element.prototype.attachShadow).run('shadow ' +
```

Comment:
Note that this now checks whether shadow DOM is supported in the test runner window rather than the "realWin".

Reply:
Given that this is a nested `describe` within a `describes.realWin`, will it not have the same effect? (From actually running the test, It seemed to me that we do have the desired skipping behavior.)

Reply:
Most likely fine unless we want to do something weird in `describes` later.